### PR TITLE
KEYCLOAK-11504 Add labels and envs from standalone image

### PR DESCRIPTION
**Pull request by hmlnarik**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -7,6 +7,12 @@ from: "ubi8-minimal:latest"
 labels:
     - name: "com.redhat.component"
       value: "redhat-sso-7-sso74-openshift-rhel8-container"
+    - name: "org.jboss.product"
+      value: &product "sso"
+    - name: "org.jboss.product.version"
+      value: &product_version "7.4.0.GA"
+    - name: "org.jboss.product.sso.version"
+      value: *product_version
```
Comment (iankko) [inline image.yaml:15]: Should the `"org.jboss.product.openjdk.version":` label be added too?

File: image.yaml [hunk 2]
```diff
@@ -18,6 +24,13 @@ labels:
     - name: "io.openshift.s2i.scripts-url"
       value: "image:///usr/local/s2i"
 envs:
+    - name: "JBOSS_PRODUCT"
+      value: *product
+    - name: "JBOSS_SSO_VERSION"
+      value: *product_version
+    - name: "PRODUCT_VERSION"
+      value: *product_version
+
```
Comment (iankko) [inline image.yaml:33]: Similarly for these -- should the:

- `JAVA_HOME`,
- `JAVA_VENDOR`,
- `JAVA_VERSION`,
- `JBOSS_CONTAINER_OPENJDK_JDK_MODULE`,
- `JBOSS_EAP_VERSION`,
- `JBOSS_IMAGE_NAME`,
- `JBOSS_IMAGE_VERSION`, 
- `JBOSS_CONTAINER_JAVA_PROXY_MODULE`, 
-`JBOSS_CONTAINER_JAVA_JVM_MODULE`,
- `JBOSS_CONTAINER_MAVEN_35_MODULE`, 
- `MAVEN_VERSION`,
- `JBOSS_CONTAINER_UTIL_LOGGING_MODULE`,
- `JBOSS_CONTAINER_MAVEN_DEFAULT_MODULE`,
- `S2I_SOURCE_DEPLOYMENTS_FILTER`,
- `JBOSS_CONTAINER_S2I_CORE_MODULE`,
- `JBOSS_CONTAINER_EAP_S2I_MODULE`,
- `DEFAULT_ADMIN_USERNAME`,
- `JBOSS_MODULES_SYSTEM_PKGS`

be defined too?

_Disclaimer_: At the moment unable to build the image for testing locally neither using docker nor podman engines, so unable to verify if EAP / JDK layers would bring those variables as an intermediary result already. But at least the `DEFAULT_ADMIN_USERNAME` & `DEFAULT_ADMIN_USERNAME` seem to be used by the EAP layer for configuration, unsure about the others.

From consistency PoV it seems to me those env vars should be preserved in new RH-SSO release stream too AFAICT.

File: image.yaml (hunk 1 above)
Comment (hmlnarik) [reply image.yaml:15]: No, that one defined is in the sso-jdk  module

Comment (iankko) [reply image.yaml:15]: Ok, failr enough. Thanks

File: image.yaml (hunk 2 above)
Comment (iankko) [reply image.yaml:33]: Per discussion with @hmlnarik  env vars listed above should be defined / provided by modules listed in the image.

**Pull request merged**